openlibrary/plugins/upstream/tests/test_account.py

```diff
@@ -1,7 +1,20 @@
+# -*- coding: utf-8 -*-
+
 from .. import account
 import web
+import os
 import re
 import pytest
+import sys
+
+
+def open_test_data(filename):
+    """Returns a file handle to file with specified filename inside test_data directory.
+    """
+    root = os.path.dirname(__file__)
+    fullpath = os.path.join(root, 'test_data', filename)
+    return open(fullpath, mode='rb')
```

Comment:
I still believe that is not a context manager so the file will not be closed but given that this is only run in short lived tests, this can pass.